### DISCUSSION TITLE
How can I display my avatar

### DISCUSSION THREAD

**Discussion by sonnh97** [dillonzq/LoveIt#824] [Q&A] [answered]
I have configured as follows:
[params.home]
    # LoveIt NEW | 0.2.0 amount of RSS pages
    rss = 10
    # Home page profile
    [params.home.profile]
      enable = true
      # Gravatar Email for preferred avatar in home page
      gravatarEmail = ""
      # URL of avatar shown in home page
      avatarURL = "/images/avatar2.jpg"

But .. result:
![Screenshot from 2023-05-10 01-14-29](https://github.com/dillonzq/LoveIt/assets/20607438/19720db5-58dd-44dc-8ddc-cfc98f60787c)

**Comment by owl4ce** [comment, marked as answer]
Short answer: Your avatar image-file should be placed in `static` directory. Hope it helps.

**Reply by sonnh97** [reply]
oh wow, tks bro. ll try it right away. It seems like this isn't on the documentation, right?

**Reply by owl4ce** [reply]
According to [the docs](https://hugoloveit.com/theme-documentation-basics/#2-installation), the documentation itself is to help people initialize their new website. Be specific to the [gohugo.io](https://gohugo.io/content-management/static-files/) docs for details.

**Reply by owl4ce** [reply]
Ah, just found it.

https://hugoloveit.com/theme-documentation-content/#contents-organization

**Reply by sonnh97** [reply]
So I will need to create a folder named 'image' in the 'static' directory

**Reply by owl4ce** [reply]
*images

**Reply by sonnh97** [reply]
Oke bro. Tks very much!!
